src/utils/isAfterDay.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+
+export default function isAfterDay(a, b) {
```

Comment:
it seems unfortunate to have both isAfterDay and isBeforeDay - would it be worth it to make one of the implementations (say this one) into `return !isBeforeDay(a, b) && !isSameDay(a, b);`?